docs-src/real_time_messaging.rst

```diff
@@ -47,6 +47,33 @@ If there was a problem connecting an error will be returned, including a descrip
     }
   }
 
+rtm.start vs rtm.connect
+---------------------------
+
+If you expect your app to be used on large teams, we recommend using starting the RTM client with `rtm.connect` rather than `rtm.start`, which is the default connection method for this client.
```

Comment:
recommedation: `... rather than the default connection method for this client, rtm.start.`